server/tests/index.test.js

```diff
@@ -57,22 +47,10 @@ describe('init', () => {
     expect(mocks.db.connect.mock.calls.length).toBe(1);
   });
 
-  it('should succeed when a valid request is made ', async () => {
+  it('should throw err if authourization is unsucessful', async () => {
     const { app } = await require('../index');
     await getResponse(query, app).then(response => {
-      expect(response.statusCode).toBe(200);
-      expect(response.body.data.__schema).toBeTruthy();
-    });
-  });
-
-  it('should fail when an invalid request is made ', async () => {
-    const { app } = await require('../index');
-    await getResponse(
-      `\n  query {\n    __schema1 {\n      queryType {\n        fields {\n          name\n        }\n      }\n    }\n  }\n  `,
-      app
-    ).then(response => {
-      expect(response.statusCode).toBe(400);
-      expect(response.body.errors).toBeTruthy();
```

Comment:
We are missing these tests 

-   'should fail when an invalid request is made '
-   'should succeed when a valid request is made '